### DISCUSSION TITLE
How can I replace my default python install with python 3.10 ?

### DISCUSSION THREAD

**Discussion by Va1a** [Homebrew/discussions#3027] [Everyday usage] [answered]
Currently, the python in /opt/homebrew/bin/python3 is python@3.9
I'd like for it to be python@3.10 which is in /opt/homebrew/opt/python@3.10/bin/python3
How can I move 3.10 out of the second layer opt directory into the main bin folder? Do I need to uninstall python3 and then reinstall python@3.10 ?

**Comment by SMillerDev** [comment, marked as answer]
No, you will need to wait until homebrew makes 3.10 default. Until then `brew info python@3.10` will tell you what you can do to make it more generally available.
